### ISSUE TITLE
Download RC version reports success, but acutally fails

### ISSUE THREAD

**Issue by codedge** [shivammathur/setup-php#601]
**Describe the bug**
I have the following step in my workflow:

```
- name: Setup PHP
  uses: shivammathur/setup-php@v2
  with:
    php-version: 8.1
    tools: deployer/deployer:v7.0.0-rc.8
  env:
    fail-fast: true
    COMPOSER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

I specified a release candidate for the PHP deployment tool [deployer](https://github.com/deployphp/deployer). The workflow reports a success, see here

<img width="757" alt="grafik" src="https://user-images.githubusercontent.com/4409904/171054353-d16f2aca-29b3-4b96-a564-27bbb883ef26.png">

.. already strange that the `8` is missing at the end. If I run `ls -ls /usr/bin/local` afterwards, there is no `deployer` inside. 

**Version**
- [ ] I have checked releases, and the bug exists in the latest patch version of `v1` or `v2`.
- [x] `v2`
- [ ] `v1`

**Runners**

- [x] GitHub Hosted
- [ ] Self Hosted

**Operating systems**
`ubuntu-latest`

**PHP versions**
8.1

**To Reproduce**

**Expected behavior**
I would expect a file `/usr/local/bin/deployer` to be present after the step in the workflow reported success.

**Comment by shivammathur** [comment]
@codedge 
Thanks for reporting this.

This has been fixed in https://github.com/shivammathur/setup-php/commit/74d43be8a37b5562a232e4b97ce460f8e0a4bd04
The fix is in the develop branch right now, it will be in the next release.

**Comment by codedge** [comment]
@shivammathur Is there an approx. release date for the next release?

**Comment by shivammathur** [comment]
I will create a release tomorrow.

**Comment by shivammathur** [comment]
Released 2.19.1 with the fix
https://github.com/shivammathur/setup-php/releases/tag/2.19.1

**Comment by shivammathur** [comment]
The executable is named `deployer` without the `phar` extension.

On ubuntu.
If you specify `deployer/deployer`. it will be installed using composer and will be in `/home/runner/.composer/vendor/bin`.
If just specify `deployer`, it will install the `phar` which will be in `/usr/local/bin`.
Both directories should be in Path. So you can just run `deployer` directly.  

Test workflow: https://github.com/shivammathur/test-setup-php/actions/runs/2447245284/workflow
